src/System.Runtime.Intrinsics/src/MatchingRefApiCompatBaseline.txt

```diff
@@ -0,0 +1,17 @@
+Compat issues with assembly System.Runtime.Intrinsics:
+CannotRemoveAttribute : Attribute 'System.CLSCompliantAttribute' exists on 'System.Runtime.Intrinsics.X86.Aes' in the implementation but not the reference.
```

Comment:
Is `CLSCompliantAttribute` meaningful? If so, we should just add it...

Reply:
It needs to be consistent.  I believe the problem is that CoreLib sets CLSCompliant on assembly level then turns if off on individual types.  Your reference assembly sets it as false on the assembly so it doesn't need to disable it on individual types. https://github.com/dotnet/corefx/blob/d81fc3589a166ab78096dee0a4ddf0cd2fbd17d7/src/System.Runtime.Intrinsics/ref/System.Runtime.Intrinsics.csproj#L4